boards/x86/qemu_x86/qemu_x86.dts

```diff
@@ -49,7 +49,8 @@
 	pcie0: pcie0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
-		compatible = "intel,pcie";
+		compatible = "pcie-controller";
```

Comment:
so this change should go in the previous patch that remove "intel,pcie"

Reply:
Which commit you referring ? This commit is specifically for board configuration changes

Reply:
> Which commit you referring ? This commit is specifically for board configuration changes

I think @tbursztyka is referring to the commit where you remove the intel,pcie binding (yaml) file. Once that file is removed there shouldn't be any more references to "intel,pcie" in dts files or the documentation.

Reply:
> > Which commit you referring ? This commit is specifically for board configuration changes
> 
> I think @tbursztyka is referring to the commit where you remove the intel,pcie binding (yaml) file. Once that file is removed there shouldn't be any more references to "intel,pcie" in dts files or the documentation.

updated